tests/daemonize/main.c

```diff
@@ -0,0 +1,46 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "daemonize.h"
+#include "warnp.h"
+
+int
+main(int argc, char * argv[])
+{
+	const char * pidfile;
+
+	/*
+	 * This will allocate some memory for the program name, which will be
+	 * free'd by an atexit() function.  The non-daemonized fork of this
```

Comment:
"free'd"?

Reply:
No?  But it highlights the function name!  I suppose that I should have written "`free()`d", though...

... ok, changed to "freed".